tauv_common/scripts/thruster_allocator.py

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env python
```

Comment:
This file should be moved into a folder on the source tree, and a short script should be created to launch it. see: http://docs.ros.org/jade/api/catkin/html/howto/format2/installing_python.html